nuget/NUnitVisualStudioTestAdapter.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>NUnitTestAdapter</id>
-        <version>${package.version}</version>
+        <version>$version$</version>
         <title>NUnit Test Adapter for VS2012 and VS2013</title>
```

Comment:
Also needs to state VS2015,  perhaps change to just say Visual Studio, and add the >=VS2012 to the description ?